File: src/main/java/net/theevilreaper/vulpes/api/repository/FontRepository.java
package net.theevilreaper.vulpes.api.repository;

import io.micronaut.data.annotation.Repository;
import io.micronaut.data.repository.PageableRepository;
import net.theevilreaper.vulpes.api.model.FontEntity;

import java.util.UUID;

@Repository
public interface FontRepository extends PageableRepository<FontEntity, UUID> {
}

File: src/main/kotlin/net/theevilreaper/vulpes/api/repository/FontRepository.kt
package net.theevilreaper.vulpes.api.repository

import net.theevilreaper.vulpes.api.model.FontModel
import org.springframework.data.mongodb.repository.MongoRepository


/**
 * @author theEvilReaper
 * @version 1.0.0
 * @since
 **/

interface FontRepository : MongoRepository<FontModel, String>